js/plates-model/rock-properties.ts

```diff
@@ -25,6 +26,12 @@ export interface IRockProperties {
 }
 
 export const ROCK_PROPERTIES: Record<Rock, IRockProperties> = {
+  [Rock.ContinentalSediment]: {
+    label: "Continental Sediments",
+    orderIndex: 0,
```

Comment:
Guessing at the order here, but presumably continental and oceanic sediments don't interact much. If they were to overlap, we'll put continental on top.